tests/filter-headers.js

```diff
@@ -7,14 +7,18 @@ describe('filter-headers', () => {
         'please-kill-me': '0',
         'accept-language': '1',
         referer: '2',
-        'user-agent': '3'
+        'user-agent': '3',
+        'x-request-uri': 'localhost',
+        'x-request-host': '/example'
```

Comment:
@kevinsimper guess host's and uri's value should be swapped?

Reply:
Nope, good spottet! 👍